### ISSUE TITLE
Markdown.TransformHtml() unexpectedly stopped

### ISSUE THREAD

**Issue by cnblogs-dudu** [tpetricek/FSharp.Formatting#355]
Unable to transform the markdown text in the following C# code  to html.
```csharp
namespace CNBlogs.Markdown.Tests
{
    public class FSharpMarkdownTest
    {
        [Fact]
        public void TransformHtmlTest()
        {
            var markdownText = @"- controller

 > demo
 > test。
>
>，";
            var html = FSharp.Markdown.Markdown.TransformHtml(markdownText);
            Console.WriteLine(html);
        }
    }
}
```
Get the following error:
```
Test 'CNBlogs.Markdown.Tests.FSharpMarkdownTest.TransformHtmlTest' failed: System.Exception : Unexpectedly stopped!
[""; "，"]
	at FSharp.Markdown.Parser.parseParagraphs@639-12.Invoke(String message)
	at Microsoft.FSharp.Core.PrintfImpl.go@523-3[b,c,d](String fmt, Int32 len, FSharpFunc`2 outputChar, FSharpFunc`2 outa, b os, FSharpFunc`2 finalize, FSharpList`1 args, Int32 i)
	at Microsoft.FSharp.Core.PrintfImpl.run@521[b,c,d](FSharpFunc`2 initialize, String fmt, Int32 len, FSharpList`1 args)
	at <StartupCode$FSharp-Core>.$Reflect.Invoke@720-4.Invoke(T1 inp)
	at FSharp.Markdown.Parser.parseParagraphs@568.Invoke(Unit unitVar)
	at Microsoft.FSharp.Collections.SeqModule.mkDelayedSeq@835.Invoke(Unit unitVar0)
	at Microsoft.FSharp.Collections.SeqModule.ToList[T](IEnumerable`1 source)
	at FSharp.Markdown.Parser.parseParagraphs@568.Invoke(Unit unitVar)
	at Microsoft.FSharp.Collections.SeqModule.mkDelayedSeq@835.Invoke(Unit unitVar0)
	at Microsoft.FSharp.Collections.SeqModule.Append@1061-1.Invoke(Unit unitVar0)
	at Microsoft.FSharp.Collections.Generator.Microsoft-FSharp-Collections-Generator-Generator-1-get_Apply@386.Invoke(Unit unitVar0)
	at Microsoft.FSharp.Collections.Generator.appG[a](Generator`1 g)
	at Microsoft.FSharp.Collections.Generator.EnumeratorWrappingLazyGenerator`1.System-Collections-IEnumerator-MoveNext()
	at Microsoft.FSharp.Collections.SeqModule.ToList[T](IEnumerable`1 source)
	at FSharp.Markdown.Markdown.Parse(String text, String newline)
	at FSharp.Markdown.Markdown.TransformHtml(String text, TextWriter writer, String newline)
	at FSharp.Markdown.Markdown.TransformHtml(String text, String newline)
	FSharpMarkdownTest.cs(21,0): at CNBlogs.Markdown.Tests.FSharpMarkdownTest.TransformHtmlTest()
```

**Comment by theprash** [comment]
I've narrowed this down to the smallest failing test case:

```fsharp
Markdown.TransformHtml ">
a"

System.Exception: Unexpectedly stopped!
[""; "a"; ""]
```

So this happens when the first line of a blockquote is empty and the next immediate line has some content. Based on http://spec.commonmark.org/dingus/ we should output this:

```html
<blockquote>
</blockquote>
<p>a</p>
```

**Comment by theprash** [comment]
Fixed in master (pull #400)
